src/khoj/search_type/text_search.py

```diff
@@ -110,7 +110,7 @@ async def query(
     file_type = search_type_to_embeddings_type[type.value]
 
     query = raw_query
-    search_model = await sync_to_async(get_user_search_model_or_default)(user)
+    search_model = await sync_to_async(get_default_search_model)()
```

Comment:
Wouldn't this be problematic for users currently using a non-default search model until the management command is successfully run to convert their entry embeddings to the default search model?

It maybe easier to avoid this issue if:
- The entry embeddings were converted via a migration to the server default search model from user default search model 
- Or switch to using server search model in two steps.
   1. Migrate the user model and embeddings to the default search model (instead of user search model) via management command
   2. Now switch to always use the server search model instead of user search model

Reply:
Fair point. This would fail for those users become the vector dimensions would be mismatched. Updated to continue using the user setting until migration script is run.